alacritty_terminal/src/renderer/mod.rs

```diff
@@ -96,7 +96,7 @@ impl ::std::fmt::Display for Error {
         match *self {
             Error::ShaderCreation(ref err) => {
                 write!(f, "There was an error initializing the shaders: {}", err)
-            },
+            }
```

Comment:
There are a few formatting regressions that were introduced in this PR, these should be fixed.

Reply:
Oh, whoops, looks like rustfmt didn't see the config file